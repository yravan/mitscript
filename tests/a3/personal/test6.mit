a = "
multiline
";

print(a);

a = "newline follows\n\ttab\" after \\";
print(a);

a = {};

a.x = true;

b = a;

b.y = false;

print(a);

a = {};
a.b = {};
a.b.c = true;
print(a);