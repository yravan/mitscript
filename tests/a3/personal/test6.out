
multiline

newline follows
	tab" after \
{x:true y:false }
{b:{c:true } }
